src/lib/fft.jl

```diff
@@ -0,0 +1,57 @@
+
+using FFTW
+using FillArrays
+# FFTW functions do not work with FillArrays, which are needed
+# for some functionality of Zygote. To make it work with FillArrays
+# as well, overload the relevant functions
+FFTW.fft(x::Fill) = FFTW.fft(collect(x))
```

Comment:
You can use `dims...` to make this more compact.

Reply:
thanks for the tip, I changed to dims... now